public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -303,18 +300,16 @@ table(width="100%")
         &lt;input ng-model="vm.favoriteHero"/>
       :marked
         In Angular&nbsp;1, the `ng-model` directive binds a form control to a property in the controller associated with the template.
-        This provides **two-way binding** whereby any changes made to the value in the view is synchronized with the model and
-        any changes to the model are synchronized with the value in the view.
+        This provides **two-way binding**, whereby any change made to the value in the view is synchronized with the model, and any change to the model is synchronized with the value in the view.
     td
       :marked
         ### ngModel
       +makeExample('cb-a1-a2-quick-reference/ts/app/movie-list.component.html', 'ngModel')(format="." )
       :marked
-        In Angular&nbsp;2, **two-way binding** is denoted with [()], descriptively referred to as a "banana in a box".
-        This syntax is a short-cut for defining both property binding (from the component to the view)
-        and event binding (from the view to the component), thereby giving us two-way binding.
+        In Angular&nbsp;2, **two-way binding** is denoted by [()], descriptively referred to as a "banana in a box". This syntax is a shortcut for defining both property binding (from the component to the view)
```

Comment:
Should the banana in a box syntax be inside backticks?

So this:

```
[()]
```

Would be this:
```
`[()]`
```